rewrite-java/src/main/java/org/openrewrite/java/JavadocPrinter.java

```diff
@@ -397,8 +397,13 @@ public J visitFieldAccess(J.FieldAccess fieldAccess, PrintOutputCapture<P> p) {
         @Override
         public J visitMemberReference(J.MemberReference memberRef, PrintOutputCapture<P> p) {
             beforeSyntax(memberRef, Space.Location.MEMBER_REFERENCE_PREFIX, p);
-            visit(memberRef.getContaining(), p);
-            visitLeftPadded("#", memberRef.getPadding().getReference(), JLeftPadded.Location.MEMBER_REFERENCE_NAME, p);
+            Expression containing = memberRef.getContaining();
+            if (containing != null) { // Invalid references will have a null containing
+                visit(containing, p);
+                visitLeftPadded("#", memberRef.getPadding().getReference(), JLeftPadded.Location.MEMBER_REFERENCE_NAME, p);
+            } else {
+                visitLeftPadded(null, memberRef.getPadding().getReference(), JLeftPadded.Location.MEMBER_REFERENCE_NAME, p);
```

Comment:
These `visitLeftPadded` might be a little too hyper focused on the failing test that was provided, but still I think it's common enough to keep this as is here. If we were to unconditionally use a `#` prefix, then that would be another print idempotency issue.